integration-tests/test_connection.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import conftest
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
```

Comment:
none of the tests actually need to register with `insights-client`, however they need to use a valid connection to Insights, and thus the certificate-based authentication

since IMHO it can be not clear in the long term to know why these test register with sub-man even if no actual registration with `insights-client`, then I recommend to add a comment before this mark to explain the situation